packages/blade/src/components/Input/BaseInput/baseInputConfig.ts

```diff
@@ -9,15 +10,65 @@ export const TAG_GAP = spacing['3'];
 /**
  * 36px
  */
-export const BASEINPUT_DEFAULT_HEIGHT: number = size['36'];
+export const baseInputHeight: Record<
+  NonNullable<BaseInputProps['size']>,
+  typeof size[keyof typeof size]
+> = {
+  medium: size['36'],
+  large: size['48'],
+} as const;
```

Comment:
Don't need to put `as const` if we are defining the type explicitly. 

Either use as const or explicit types